tests/Doctrine/Tests/Common/Cache/SQLite3CacheTest.php

```diff
@@ -30,6 +31,28 @@ public function testGetStats()
         $this->assertNull($this->_getCacheDriver()->getStats());
     }
 
+    public function testFetchSingle()
+    {
+        $id = uniqid('sqlite3_id_');
+
+        $data = $this->_getCacheDriver();
+
+        $this->_getCacheDriver()->save($id, $data, 30);
+
+        try {
+            $actual = $this->_getCacheDriver()->fetch($id);
+
+        } catch (\PHPUnit_Framework_Error $e) {
+            $this->fail('Unexpected exception has been raised. ' . $e->getMessage());
```

Comment:
Don't catch ``\PHPUnit_Framework_Error`` to make it fail again. PHPUnit already mark the testsuite as failed in case an error occurs. You don't need to do it yourselves.